sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java

```diff
@@ -237,6 +361,13 @@ public EventDirectorySteps(
           webDriverHelpers.clickOnWebElementBySelector(APPLY_FILTERS_BUTTON);
           TimeUnit.SECONDS.sleep(10);
         });
+    When(
+        "I hover to Event Groups column of the Event result",
```

Comment:
what is the need for this hover?

Reply:
It is required in test case to hover to that localization.